Roslyn.sln

```diff
@@ -401,6 +401,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Insertion", "Insertion", "{
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevDivInsertionFiles", "src\Setup\DevDivInsertionFiles\DevDivInsertionFiles.csproj", "{6362616E-6A47-48F0-9EE0-27800B306ACB}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.VisualStudio.LanguageServices.FSharp", "src\VisualStudio\FSharp\Microsoft.VisualStudio.LanguageServices.FSharp.csproj", "{68624359-92E2-4FC5-AF9E-D65ABC3A06DC}"
```

Comment:
💭 I would prefer this follow the **ExternalAccess** naming and separation seen in #31749

Reply:
@jinujoseph @tmat @jasonmalinowski @sharwell @dotnet/roslyn-ide are we all agree with the ExternalAccess naming @sharwell is pushing?

I don't have any preference as long as people are all in agreement. if all agree, I will create new issue saying we should move existing shim project to same naming pattern and change this one.

Reply:
I think we should hold of on naming this any specific.

Reply:
@jinujoseph My intent is to avoid confusion between the adapter assemblies (which are not intended for public consumption) and all of our various assemblies that have public APIs. Consistent naming and placement of these assemblies will allow us to more efficiently address the underlying IVT problem without adding confusion.